plugins/modules/win_scoop.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Jamie Magee <jamie.magee@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: win_scoop
+short_description: Manage packages using Scoop
+description:
+- Manage packages using Scoop.
+- If Scoop is missing from the system, the module will install it.
+options:
+  arch:
+    description:
+    - Force Scoop to install the package of a specific process architecture.
+    type: str
+    choices: [ 32bit, 64bit ]
+  global:
+    description:
+    - Install the app globally
+    type: bool
+    default: no
+  independent:
+    description:
+    - Don't install dependencies automatically
+    type: bool
+    default: no
+  name:
+    description:
+    - Name of the package(s) to be installed.
+    type: list
+    elements: str
+    required: yes
+  no_cache:
+    description:
+    - Don't use the download cache
+    type: bool
+    default: no
+  purge:
+    description:
+    - Remove all persistent data
+    type: bool
+    default: no
+  skip:
```

Comment:
Skip is quite generic here, should be more specific like `skip_checksum` or `checksum_check` in case we need `skip` for some future purpose.

Reply:
In this case, I think `skip` is just a bit too vague. To me it pretty much says skip this package which could be conditional for any reason. While modules should closely reflect the underlying resource it manages we should try and ensure we keep some consistency with existing Ansible conventions as well as paper over some of the more confusing aspects of a resource. In this case `skip_checksum` tells the Ansible author what exactly it is skipping whereas seeing `skip` in a playbook is only really obvious to people who have used scoop before.